server/app/models/template.js

```diff
@@ -69,9 +69,9 @@ class TemplateUnit extends Model {
                 // eslint-disable-next-line
                 get() {
                     const startAt = this.getDataValue('startAt');
-
+                    const day = startAt.getDay() === 0 ? 7 : startAt.getDay();
```

Comment:
Can be simplified to `const day = startAt.getDay() || 7`

Reply:
Oh wonderfull :)